stable/traefik/README.md

```diff
@@ -98,6 +98,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `rbac.enabled`                  | Whether to enable RBAC with a specific cluster role and binding for Traefik | `false`                            |
 | `nodeSelector`                  | Node labels for pod assignment                                       | `{}`                                      |
 | `tolerations`                   | List of node taints to tolerate                                      | `[]`                                      |
+| `proxyProtocol.enabled`         | Enable receiving the Proxy protocol instead http or https            | `false`                                   |
+| `proxyProtocol.trustedIPs`      | List of trusted ip (ranges) from which to allow Proxy protocol traffic | `["10.0.0.0/8"]`                        |
```

Comment:
I would reword the descriptions above for better clarity...

Propose changing

> Enable receiving the Proxy protocol instead http or https

to just:

> Enable PROXY protocol support.

Maybe add a link to the section where you elaborate on it?

Propose changing

> List of trusted ip (ranges) from which to allow Proxy protocol traffic

to

> List of proxy IPs (CIDR ranges) trusted to accurately convey the end-user IP.

Reply:
The default of `10.0.0.0/8` isn't going to work for lots and lots of people. It might be better to _not_ have a default and make chart install fail if it's not defined. This can be done with the some conditional logic and the [fail function from Sprig](http://masterminds.github.io/sprig/flow_control.html) (which Helm supports).